packages/subgraph/tests/multisig/multisig.test.ts

```diff
@@ -180,33 +216,33 @@ test('Run Multisig (handleApproved) mappings with mock event', () => {
   handleApproved(event);
 
   // checks
-  const member = Address.fromString(ADDRESS_ONE).toHexString();
-  const pluginId = Address.fromString(CONTRACT_ADDRESS).toHexString();
-  const memberId = pluginId + '_' + member;
+  const memberAddress = Address.fromString(ADDRESS_ONE);
+
+  const memberEntityId = generateMemberEntityId(pluginAddress, memberAddress);
 
+  const voterEntityId = generateVoterEntityId(memberEntityId, proposal.id);
```

Comment:
I wonder if we should use the wording of "approver" here?
I know the inside the pluginEnitities we call them members,
while we are assigning `MultisigApprover` to them, in this example.

```
type MultisigPlugin implements IPlugin @entity {
  "TODO: attributes should be appended to Plugin once plugin is seperated from Aragon OSx"
  id: ID!
  dao: Dao!
  pluginAddress: Bytes!

  installations: [PluginInstallation!]! @derivedFrom(field: "plugin")

  proposalCount: BigInt
  proposals: [MultisigProposal!]! @derivedFrom(field: "plugin")
  members: [MultisigApprover!]! @derivedFrom(field: "plugin")

  # multisig settings
  minApprovals: Int
  onlyListed: Boolean
}

type MultisigApprover @entity {
  id: ID! # plugin_address + member_address
  address: String # address as string to facilitate filtering by address on the UI
  proposals: [MultisigProposalApprover!]! @derivedFrom(field: "approver")
  plugin: MultisigPlugin!
}
```

but this pattern repeats for other plugins
i think we should use "approver" in our test to reduce confutions, since the member atribute is ment for the consumers

@Michael-A-Heuer i would like to have a second opinion on this

Reply:
I mean the tests themselves are not clear but i think its a matter of how this is structured and named

```ts
 // checks
  const memberAddress = Address.fromString(ADDRESS_ONE);

  const memberEntityId = generateMemberEntityId(pluginAddress, memberAddress);

  const voterEntityId = generateVoterEntityId(memberEntityId, proposal.id);
  // check proposalVoter
  assert.fieldEquals(
    'MultisigProposalApprover',
    voterEntityId,
    'id',
    voterEntityId
  );
  assert.fieldEquals(
    'MultisigProposalApprover',
    voterEntityId,
    'approver',
    memberEntityId
  );
  assert.fieldEquals(
    'MultisigProposalApprover',
    voterEntityId,
    'proposal',
    proposal.id
  );
  ```

If you look at this  you have `memberEntityId` and `voterEntityId`, here `voterEntityId` is the id for the `MultisigProposalApprover` so we shoyld change the name of this to `proposalApproverEntityId` or something similar, but if you go a bit down, `voterEntityId`  is the value of the field  `approver` inside the `MultisigProposalApprover` so what will be the name for this id?, `approverEntityId`?

Reply:
Puh, I think we should consistently name things and as you both point out, things seem to be not consistently named. 
This sounds like more work and could be worth a separate task.